modules/ROOT/pages/afm-modify-schedule.adoc

```diff
@@ -23,25 +21,22 @@ There are two different methods for modifying a schedule.
 To modify a schedule from the *Schedules* section of a monitor:
 
 . Select the monitor in which you want to modify a schedule.
-. Click the *Schedules* tab.
-. Find the schedule in the list, click the three dots on the right end of the table row, and select *Modify*.
-. In the Modify Schedule dialog, change one or more of these items:
+. Find the schedule in the list, tap *More* (image:afm-ui-more-button.png[width=0.75%,height=0.75%]) and select *Modify Schedule*.
+. In the *Modify Schedule* dialog, change one or more of the following:
 +
-* The test version that you want the schedule to run
-* The location to run the test from
+* The location at which to run the test
 * The frequency at which to run the test
 . Click *Save*.
 
-To add a schedule from the list of schedules that use a particular location:
+To modify a schedule from the list of schedules using a particular location:
```

Comment:
What or who is "using a particular location"? You? A schedule? The list of schedules? Clarify.

Reply:
I removed "using a particular location" as it doesn't make sense here. The steps below specify the particular location, so I think we're good.